rj_gameplay/rj_gameplay/gameplay_node.py

```diff
@@ -149,12 +149,12 @@ def set_play_state(self, play_state: msg.PlayState):
     def set_match_state(self, match_state: msg.MatchState):
         self.match_state = match_state
 
-    def debug_callback(self, play: stp.play.IPlay, skills):
+    def debug_callback(self, play: stp.play.IPlay, tactics: list):
```

Comment:
can you add a docstring that explains what this does now?

also, as we discussed recently, you can actually have the play repr() call the repr() of each tactic if you want, instead of passing in a list of tactics

Reply:
(but that can be a later improvement)